### PR TITLE
China / Germany Cloud Fix

### PR DESCRIPTION
**Pull request by JustinMDotNet**
Added missing resources to chinaAzureSettings and germanyAzureSettings in providerSettings

This PR fixes #

### CODE REVIEW

File: extensions/azurecore/src/account-provider/providerSettings.ts [hunk 1]
```diff
@@ -220,7 +220,12 @@ const germanyAzureSettings: ProviderSettings = {
 				endpoint: 'https://vault.microsoftazure.de',
 				azureResourceId: AzureResource.AzureKeyVault
 			},
-			redirectUri: 'https://vscode-redirect.azurewebsites.net/'
+			redirectUri: 'https://vscode-redirect.azurewebsites.net/',
+			scopes: [
+				'openid', 'email', 'profile', 'offline_access',
+				'https://management.microsoftazure.cn/user_impersonation'
```
Comment (Charles-Gagnon) [inline extensions/azurecore/src/account-provider/providerSettings.ts:226]: This appears to be using the china endpoint which doesn't seem correct

Comment (JustinMDotNet) [reply extensions/azurecore/src/account-provider/providerSettings.ts:226]: Great catch. Copy paste got me!

File: extensions/azurecore/src/account-provider/providerSettings.ts [hunk 2]
```diff
@@ -245,9 +255,14 @@ const chinaAzureSettings: ProviderSettings = {
 			},
 			armResource: {
 				id: SettingIds.arm,
-				endpoint: 'https://managemement.chinacloudapi.net',
+				endpoint: 'https://management.chinacloudapi.cn',
```
Comment (JustinMDotNet) [inline extensions/azurecore/src/account-provider/providerSettings.ts:258]: Fixed spelling

**Review by Charles-Gagnon — APPROVED** [review]

**Review by kburtram — APPROVED** [review]
LGTM.  I assume you got these new settings from somewhere "official"

**Comment by JustinMDotNet** [comment]
> LGTM. I assume you got these new settings from somewhere "official"

Haha yes, the URLs came from here:
https://docs.microsoft.com/en-us/azure/china/resources-developer-guide

**Comment by zhengyiqun** [comment]
Linux version 1.31.1, still unable to add azure china account. The error is: "The resource principal named https://management.core.windows.net/ was not found in the tenant named *.partner.onmschina.cn"

**Pull request merged**